src/eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -439,7 +439,7 @@ def fitlc(eventlabel, ecf_path=None, s4_meta=None, input_meta=None):
                                     "_Meta_Save"), save=[])
             else:
                 for channel in range(chanrng):
-                    log.writelog(f"\nStarting Channel {channel+1} of "
+                    log.writelog(f"\nStarting Channel {channel} of "
                                  f"{chanrng}\n")
```

Comment:
Should this be `chanrng-1` or do we want it to finish at `Starting Channel n-1 of n`?

Reply:
I editing my comment above.  I thought you might ask about this.